suricata.yaml.in

```diff
@@ -808,6 +810,9 @@ pcap-file:
 # "detection-only" enables protocol detection only (parser disabled).
 app-layer:
   protocols:
+    krb5:
+      enabled: @rust_config_enabled@
+      warn-weak-crypto: yes
```

Comment:
Is this an app-layer option instead of just disabling the rule for performance reasons?

Reply:
Hmm, I think I missed the easiest solution. I'll remove this option, disabling the rule should be enough